mkdocs.yml

```diff
@@ -16,8 +23,9 @@ extra:
     project: Mezzio
     installation:
         config_provider_class: 'Mezzio\Session\ConfigProvider'
-    current_version: v1
+    current_version: v2
     versions:
+      - v2
       - v1
 plugins:
     - redirects:
```

Comment:
The redirects should be changed to point to the v2 equivalent - when v2 is released, we want old urls to redirect to the most recent version

Reply:
Modified redirects to go to `v2`.
Also, added `non-empty-string` to `SessionInterface::getId()` and adapted code where needed.